stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift

```diff
@@ -78,11 +78,12 @@ private class TestCluster {
     // be started when we set TFE_ContextSetServerDef.
     for i in 1..<taskCount {
       debugLog("Starting task \(i)...")
-      let serverDefText = createServerDef(forTasksConfig: tasks, withTaskIndex: i)
-      let serverDef = TFE_GetServerDef(serverDefText, status)
+      let serverDefText = createServerDef(forTasksConfig: tasks, taskIndex: i)
+      let serverDef: UnsafeMutablePointer<TF_Buffer>! = TFE_GetServerDef(
```

Comment:
You can omit redundant type information.

```suggestion
      let serverDef: UnsafeMutablePointer! = TFE_GetServerDef(
```

Reply:
done.